host/libhackrf/CMakeLists.txt

```diff
@@ -33,6 +33,11 @@ include(${PROJECT_SOURCE_DIR}/../cmake/set_release.cmake)
 add_definitions(-DLIBRARY_RELEASE="${RELEASE}")
 set(CMAKE_MODULE_PATH ${PROJECT_SOURCE_DIR}/../cmake/modules)
 
+option(DISABLE_USB_ENUMERATION "Prevent libusb from trying to enumerate devices. Useful on non-root android" OFF)
```

Comment:
I'd rather name this something like `DISABLE_USB_DEVICE_DISCOVERY`. I think it's clearer if it matches libusb's name & doesn't get confused with OS-level enumeration.